### PR TITLE
Added queries customs and resolved warnings from tests

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Added and tested queries custom and resolved  tests'warnings
## Trello Card
https://trello.com/c/MJHuvLVh/25-agregar-m%C3%A9todos-m%C3%A1s-complejos-a-los-repositorios

### CODE REVIEW

File: src/main/java/wolox/training/repositories/UserRepository.java [hunk 1]
```diff
@@ -14,4 +16,10 @@
 
     Optional<List<User>> findByBirthdateBetweenAndNameIgnoreCaseContaining(LocalDate start, LocalDate end,
             String string);
+
+    @Query("SELECT u FROM USERS u WHERE (:dateFrom IS NULL OR u.birthdate >= :dateFrom) AND (:dateTo IS NULL"
+            + " OR u.birthdate <= :dateTo) AND (:someChars IS NULL OR LOWER(u.name) LIKE CONCAT('%',LOWER(:someChars),'%'))")
+    Optional<List<User>> findUserByBirthdayBetweenAndContaining(@Param("dateFrom") LocalDate dateFrom,
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/repositories/UserRepository.java]: Same here, Optional shouldn't be used with a List

Comment (alisonreynoso) [reply src/main/java/wolox/training/repositories/UserRepository.java]: Done :D

**Review by julianvasquez-wolox — APPROVED** [review]
Lets gooo 🙌🏼

**Review by Jimenarosello — APPROVED** [review]
Nice 😄 👍

**Pull request merged**